repository/src/main/java/com/epam/esm/entities/Identifiable.java

```diff
@@ -3,11 +3,7 @@
 import lombok.AllArgsConstructor;
 import lombok.NoArgsConstructor;
 
-import javax.persistence.Column;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.MappedSuperclass;
+import javax.persistence.*;
```

Comment:
avoid using *

Reply:
Fixed.  I've forgotten to fix idea settings for imports again...